src/navsat_transform.cpp

```diff
@@ -153,7 +168,7 @@ void NavSatTransform::run()
 
       if (transform_good_ && !use_odometry_yaw_ && !use_manual_datum_) {
         // Once we have the transform, we don't need the IMU
-        imu_sub.reset(nullptr);
+        imu_sub = nullptr;
```

Comment:
this is leaking memory, please reset the smart pointer

Reply:
Thanks, good catch. Not even sure why I changed it.. ?